README.md

```
@@ -22,9 +22,17 @@ A detailed guide for multiple platforms setup can be found [in the Flutter insta
 
 ### Setup Project
 
-- Clone this repository using `git clone https://github.com/CircuitVerse/mobile-app.git`.
+- Clone this repository **with its submodules**:
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)